index.bs

```diff
@@ -4291,15 +4298,15 @@ To <dfn noexport>trigger attribution</dfn> given an [=attribution trigger=] |tri
     with |trigger|.
 1. If |trigger|'s [=attribution trigger/event-level trigger configurations=]
     [=set/is empty=] and |hasAggregatableData| is false, return.
-1. Let |matchingSources| be the result of running [=find matching sources=] with |trigger|.
-1. If |matchingSources| [=list/is empty=]:
+1. Let |matchingSourcesResult| be the result of running [=find matching sources=] with |trigger|.
```

Comment:
Let's use the multiple-assignment syntax here: https://infra.spec.whatwg.org/#variables.

```
Let (|sourceToAttribute|, |sourcesToRemove|) be the result...
```

Reply:
Updated, thanks.